client/dashboard/app/router.tsx

```diff
@@ -83,6 +83,17 @@ const sitesRoute = createRoute( {
 			queryClient.ensureQueryData( userPreferencesQuery() ),
 		] );
 	},
+	validateSearch: ( search ) => {
+		// Deserialize the view search param if it exists on the first page load.
+		if ( typeof search.view === 'string' ) {
+			try {
+				return { ...search, view: JSON.parse( search.view ) };
+			} catch ( e ) {
+				// pass
```

Comment:
If parsing fails, then perhaps we should return a search object where the `view` property has been removed?

If we had Sentry connected this is also where I'd advocating sticking a Sentry call rather than ignoring the catch completely.